config.sample.py

```diff
@@ -1,11 +1,18 @@
+# -*- coding: utf-8 -*-
+
 # database must be "se-simulator"
 db = {
     "host": "localhost",
     "user": "username",
     "passwd": "password",
 }
 
-# check http://flask.pocoo.org/docs/0.12/quickstart/#sessions on how to create
+# Check http://flask.pocoo.org/docs/0.12/quickstart/#sessions on how to create
 secret_key = 
 
+# See https://pythonhosted.org/Flask-Session/
+session_type = "filesystem"
+
+make_cacheable = True
```

Comment:
@jaytaylor 

In case you were wondering what `make_cacheable` does:
I have little experience with making sites performant on many page views because most of my websites have very little. So I was afraid that everything would break down if more than a handful people visited the website and planed on enabling nginx caching in case. And setting this to true makes sure everyone sees the same page (otherwise everyone sees what they have already upvoted)

But it turned out that this was completely unnecessary. Even with 16000 page views in 24 hours my tiny vServer didn't seem to reach more than a few percent cpu usage and never slowed down, so I never cached anything.